### DISCUSSION TITLE
Azure Active Directory login not working after update

### DISCUSSION THREAD

**Discussion by RobertSeidler** [requarks/wiki#5596] [Error / Bug Report]
Hi, 

I recently updated our wiki.js to the newest version (2.5.286). After the update the azure active directory login didn't work anymore. I was still logged in, and it still worked. But as soon as I logged out I couldn't log back in. When navigating to /login (skip login screen was activated), the site was loading for 300s and afterwards just displayed this error: 
```
502 Proxy Error 

The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request

Reason: Error reading from remote server
```

Using /login?all=1 I was still able to use local db auth, but clicking on Azure Active Directory produced the mentioned error.

I reverted back to version 2.5.275 for now where it works without problems.

**Comment by NGPixel** [comment]
There wasn't any change to azure AD in 2.5.286 compared to 2.5.285, so this is odd.

However, the error you posted is from your reverse proxy and not Wiki.js. Try bypassing your reverse proxy to get the actual error.

**Comment by RobertSeidler** [comment]
thanks for your fast response. 
Although we are using a reverse proxy for visitors from outside our company network, I'm sure I tried accessing our wiki without it (to pinpoint the problem) and at least thought, that I got the same error (although it is possibly I only assumed this, because of the same really long timeout and didn't actually wait the full 300s). 

I'll try the update again, as soon as I got a little time, and if the problem persists, I'll post the correct error (hopefully).

**Reply by NGPixel** [reply]
Error 502 is a generic reverse proxy error. It's not possible for this error to come directly from Wiki.js.
